frontend/src/firebase/firebase.json

```diff
@@ -0,0 +1,4 @@
+{
+  "apiKey": "AIzaSyBuUKIjVd1CtXa4T5FR47mLx2ew-vj7780",
```

Comment:
oop?

Reply:
is this what caused firebase to yell at us

Reply:
It's ok https://stackoverflow.com/a/37484053

Reply:
bro @XiaochenSeanYu I just saw this this is def not it smh